### PR TITLE
Test cases for CVSB-5558

### PR DESCRIPTION
**Pull request by dorinNicolae**

### CODE REVIEW

File: src/test/java/logs/PostLogs.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import net.thucydides.core.annotations.Title;
+import net.thucydides.core.annotations.WithTag;
+import net.thucydides.core.annotations.WithTags;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import steps.LogsSteps;
+
+@WithTags(
+        {
+                @WithTag(type = "TestResults", name = "All"),
```
Comment (lgvasile) [inline src/test/java/logs/PostLogs.java]: rename TestResults

File: src/main/java/steps/LogsSteps.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import io.restassured.response.Response;
+import net.thucydides.core.annotations.Step;
+
+public class LogsSteps {
+
+    LogsClient logsClient = new LogsClient();
+    Response response;
+
+    @Step
+    public void postActivities(Object logs) {
```
Comment (lgvasile) [inline src/main/java/steps/LogsSteps.java]: put Logs object as parameter

**Review by teoAlexandru — DISMISSED** [review]

File: src/main/java/steps/LogsSteps.java (hunk 2 above)
Comment (dorinNicolae) [reply src/main/java/steps/LogsSteps.java]: done

File: src/test/java/logs/PostLogs.java (hunk 1 above)
Comment (dorinNicolae) [reply src/test/java/logs/PostLogs.java]: done

**Review by teoAlexandru — APPROVED** [review]

**Review by lgvasile — APPROVED** [review]

**Pull request merged**